ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchasesHybridCommonTests.swift

```diff
@@ -10,17 +10,23 @@ import Quick
 import Nimble
 @testable import PurchasesHybridCommon
 @testable import RevenueCat
+import Foundation
 
 class PurchasesHybridCommonTests: QuickSpec {
 
-    private let mockCustomerInfo = try! CustomerInfo(data: [
-        "request_date": "2019-08-16T10:30:42Z",
-        "subscriber": [
-            "first_seen": "2019-07-17T00:05:54Z",
-            "original_app_user_id": "",
-            "subscriptions": [:],
-            "other_purchases": [:]
-        ]])
+    private var mockCustomerInfo = CustomerInfo.fromJSON(
```

Comment:
This doesn't need to be mutable right?
```suggestion
    private static let mockCustomerInfo = try! CustomerInfo.fromJSON(
```

Here you can do `try!` since this belongs to all tests, and gets created at initialization.